dlp-terraform/lambda.tf

```diff
@@ -0,0 +1,73 @@
+
+data "archive_file" "send_email_files" {
+  type        = "zip"
+  output_path = "outputs/send_email.zip"
+  source_dir  = "lambda/source-files"
+}
+
+resource "aws_lambda_function" "send_email" {
+  # If the file is not in the current working directory you will need to include a
+  # path.module in the filename.
+  filename      = "outputs/send_email.zip"
+  function_name = "send_email"
+  role          = "arn:aws:iam::521654603461:role/send_email"
```

Comment:
@thomaschin35 did u test the deploy and run the python unit tests to see if stuff works. you can also try training simple iris model to validate